src/components/Table/README.md

```diff
@@ -237,10 +237,11 @@ const MyTable1 = withTableSettings({sortable: false})(Table);
 
 ### Options
 
-| Name     | Description                                       |      Type      | Default |
-| :------- | :------------------------------------------------ | :------------: | :-----: |
-| width    | Settings' popup width                             | `number` `fit` |         |
-| sortable | Whether or not add ability to sort settings items |   `boolean`    | `true`  |
+| Name       | Description                                         |      Type      | Default |
+| :--------- | :-------------------------------------------------- | :------------: | :-----: |
+| width      | Settings' popup width                               | `number` `fit` |         |
```

Comment:
```suggestion
| width      | Settings' popup width                               | `number` `"fit"` |         |
```

Add quotes to indicate this is a string

Reply:
[done](https://github.com/gravity-ui/uikit/pull/1627/commits/e839ec965c992caa65af754928a7133cdc007b0b)